cypress/support/fragments/data_import/logs.js

```diff
@@ -44,6 +44,12 @@ const checkIsItemCreated = () => {
   }));
 };
 
+const checkIsInvoiceCreated = () => {
+  cy.do(MultiColumnListCell({ row: 0, columnIndex: 7 }).perform(element => {
```

Comment:
You can use simpler construction:

`cy.expect(MultiColumnListCell({ row: 0, column: 'Created' }).exists())`

Reply:
Done